app/views/accounts/registrations/new.html.erb

```diff
@@ -1,45 +1,63 @@
-<h2>Sign up</h2>
-
+<h1 style="line-height:1.2em">Sign up to the NZOI's training site</h1>
+<ul>
+  <li>Submit solutions to problems.</li>
+  <li>Compete in contests.</li>
+  <li>Join groups.</li>
+  <li>Access more problems.</li>
+</ul>
 <%= form_for(resource, :as => resource_name, :url => registration_path(resource_name)) do |f| %>
   <%= devise_error_messages! %>
 
-  <p><%= f.label :username %><br />
+  <h3><%= f.label :username %><br />
   <%= f.text_field :username %></p>
 
-  <p><%= f.label :name %><br />
+  <h3><%= f.label :name %><br />
   <%= f.text_field :name %></p>
 
-  <p><%= f.label :email %><br />
+  <h3><%= f.label :email %><br />
   <%= f.email_field :email %></p>
 
-  <p><%= f.label :password %><br />
+  <h3><%= f.label :password %><br />
   <%= f.password_field :password %></p>
 
-  <p><%= f.label :password_confirmation %><br />
+  <h3><%= f.label :password_confirmation %><br />
   <%= f.password_field :password_confirmation %></p>
 
-  <p><%= f.label :country_code, "Country" %><br />
+  <h3><%= f.label :country_code, "Country" %><br />
   <%= f.country_select(:country_code, selected: @user.country_code || request.location.try(:country_code), priority_countries: ["NZ", "AU"]) %></p>
 
-  <p><%= f.label :school_graduation, "School Graduation" %><br />
+  <h3><%= f.label :school_graduation, "School Graduation" %></h3>
+  <p> If you attend a primary, intermediate, or secondary school (or equivilent) in New Zealand then we ask that you please enter your estimated graduation date (the month and year you expect to finish secondary school by). You can update this later if it changes. Our website uses your graduation date to calculate your eligibility to:ssss</p>
+  <ul>
+    <li>Compete <u>officially</u> in the NZIC (New Zealand Informatics Contest).</li>
+    <li>Potentally be invited to a programming camp run by the NZOI (New Zealand Olympiad in Informatics)</li>
+  </ul>
+  <p>Also note that:</p>
+  <ul>
+    <li>Anyone is welcome to create an account and use our site without specifying a graduation date.</li>
+    <li>Anyone can compete in the NZIC unofficially without specifying a graduation date. However, you will not be ranked on offical scoreboards or be eligible to be invited to our programming camps.</li>
+    <li>Only New Zealand citizens or perminant residents, with a valid future graduation date, will be considered for our international IOI team.</li>
```

Comment:
```suggestion
    <li>Anyone can compete in the NZIC unofficially without specifying a graduation date. However, you will not be ranked on official scoreboards or be eligible to be invited to our programming camps.</li>
    <li>Only New Zealand citizens or permanent residents that have attended camp will be considered to represent New Zealand at the International Olympiad in Informatics.</li>
```

Reply:
Hmmm. I need to start typing things in something that does spell checking